segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/TLSConfigChangeEventConsumer.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.segmentstore.server.host.handler;
+
+import java.nio.file.WatchEvent;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.function.Consumer;
+
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+@RequiredArgsConstructor
+@Slf4j
+public class TLSConfigChangeEventConsumer implements Consumer<WatchEvent<?>> {
+
+    /**
+     * A counter representing the number of times this object has been asked to
```

Comment:
This comment is wrong. You refer to `this object` but this field is static. I don't think you should make it static

Reply:
Yeah, initially an instance of this class was supposed to be created for each Netty pipeline. So, the counter needed to be static. Now, we register the watcher just once, so there is no need to make this static. I've fixed this in the latest commit.